examples/kubernetes/ceph-osd-v1-ds.yaml

```diff
@@ -17,9 +17,12 @@ spec:
       nodeSelector:
         node-type: storage
       volumes:
-        - name: devices
-          hostPath:
-            path: /dev
+# Comment back in to use direct device access.
+# Note that this will also require enabling privileged access and
```

Comment:
If you don't expose `/dev/` how to you access the device to use it as an OSD? Moreover without privileged you won't be able to partition the dev etc.
How did you test this?